components/CustomAssets/AssetHistory/AssetHistory.tsx

```diff
@@ -0,0 +1,56 @@
+import { HStack, Text, Box } from '@ironfish/ui-kit'
+import { CommonTable, HashView, CopyValueToClipboard } from 'components'
+import { formatInTimeZone } from 'date-fns-tz'
+import { ColumnProps } from 'components/Table/types'
+import BurnAction from 'assets/svg/burn-action.svg'
+import MintAction from 'assets/svg/mint-action.svg'
+import { AssetDescriptionType } from 'types'
+
+const upperFirst = (str: string) =>
+  str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
+
+const COLUMNS: Array<ColumnProps<AssetDescriptionType>> = [
+  {
+    key: 'action',
+    label: 'Action',
+    render: item => (
+      <HStack alignItems="center" gap={4}>
+        {item.type === 'MINT' ? <MintAction /> : <BurnAction />}
+        <Text>{item.type ? upperFirst(item.type) : ''}</Text>
+      </HStack>
+    ),
+  },
+  {
+    key: 'quantity',
+    label: 'Quantity',
+    render: item => item.value,
+  },
+  {
+    key: 'hash',
+    label: 'Txn Hash',
+    render: item => (
+      <CopyValueToClipboard
+        value={item.transaction_hash}
+        label={<HashView hash={item.transaction_hash} />}
+      />
+    ),
+  },
+  {
+    key: 'timestamp',
+    label: 'Timestamp',
+    render: item =>
+      formatInTimeZone(item.block_timestamp, 'UTC', 'yyyy/MM/dd hh:mm:ss aa'),
+  },
+]
+
+type Props = {
+  assetHistory: Array<AssetDescriptionType>
+}
+
+export function AssetHistory({ assetHistory }: Props) {
```

Comment:
Just a question. Why simple functions?

Reply:
Mostly personal preference, but I also like that functions are hoisted, so you can put helper components at the bottom of a file.

I've seen that `React.FC` is [discouraged](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/) by the community, so using `const Foo: React.FC<Props>` isn't really an issue anymore.